components/ContentPicker/index.js

```diff
@@ -87,7 +87,7 @@ const ContentPicker = ({
 	// Run onPickChange callback when content changes.
 	useEffect(() => {
 		onPickChange(content);
-	}, [content, onPickChange]);
+	}, [content]);
```

Comment:
@tlovett1 This will likley cause other issues because of the omitted dependency. The correct approach would be making the `onPickChange` a callback using the `useCallback` hook: https://reactjs.org/docs/hooks-reference.html#usecallback

Reply:
I will defer to React guru @nicholasio on this one. I think since it's passed via prop from an unknown place, this is fine.

Reply:
I don't think `useCallback` here is the best option as it's essentially going to expect that users of this library are using it and if they aren't there will be a non-stop re-rendering issue.

The easiest way to solve this is to remove the dep and disable the eslint rule and it should be mostly fine.

However, I took a quick look at the code and a better solution to this issue is to stop using `useEffect` as a way to watch for content changes and instead use the `setContent` callback to trigger `onPickChange`.

```javascript
setContent((previousContent) => { 
  const newContent = [
	  {
		  id: item.id,
		  uuid: uuidv4(),
		  type: 'subtype' in item ? item.subtype : item.type,
	  },
	  ...previousContent,
  ];
 onPickChange(newContent)
 return newContent;
});
```
This would need to be done on every setContent call

Reply:
Got it. I'd like to leave it as it is now in the interest of changing less :)